pkgs/applications/version-management/vcsh/default.nix

```diff
@@ -1,26 +1,42 @@
-{ lib, stdenv, fetchFromGitHub, which, git, ronn, perlPackages }:
+{ lib, stdenv
+, fetchurl
+, makeWrapper
+, pkg-config
+, autoconf
+, automake
```

Comment:
@alerque also `vcsh` doesn't need autoconf and automake. I pushed a change to here as well.

Reply:
Actually it does. Not an run time of course, but that *is* how it builds (that is new in the 2.x release series).

Reply:
Hmmm, unless the parts to *run* the autoconf/automake generated bits are included in `makeWrapper` and/or `pkgconfig`. The tarball ships with a `configure` file build from `configure.ac` using `make dist`, so maybe the necessary batteries are included?

Reply:
Never mind, since a reconf is not needed to configure and build what is in the source dist taraball those shouldn't be needed, cf. https://github.com/NixOS/nixpkgs/pull/136453#issuecomment-914160411

Reply:
Yes. What I mean by "`autoconf` and `automake` not needed" is that the _build_ doesn't need them, because the `./configure` script included in the release tarball is complete.